### ISSUE TITLE
Getting it up to date with origin

### ISSUE THREAD

**Issue by Trollwut** [tryone144/compton#9]
Hi there!

Any plans on merging the newest origin compton into it?

Seems like updates are two/one years ago, so there might have been some changes. As this compton fork gets some popularity nowadays in /r/unixporn, that might be for some folks. :>

**Comment by Trollwut** [comment]
Also to add, I saw that Arch Linux' package is being built from https://github.com/yshui/compto, which is about 600 commits ahead of https://github.com/chjj/compton , as the latter one seems to be unmaintained (quote from the yshui repo)

EDIT: I also fiddled around a bit at weekend and tested your fork. The kawase blur is incredibly sexy!

**Comment by Trollwut** [comment]
Oopsie, I just saw that you already are in the progress of doing this.

For future readers: https://github.com/yshui/compton/issues/32

TL;DR kawase blur will be integrated into yshui's fork, but implementing it requires much work (reasons in the link). It's currently a WIP

Thanks for all your work!

**Comment by tryone144** [comment]
Dual-filter kawase blur has been merged into `next` on upstream picom. Currently only compatible with `--experimental-backends`.
